MJL20151028 - Patch created.

This patch merely changes the href for the old Jargon File versions
back to the absolute URL for catb.org, since I'm not interested in
mirroring those on GitHub Pages.  I do however add a link to their
location in my BitBucket repository

--- download.html	2015-10-28 08:58:33.953223754 +1100
+++ download.html.mjl	2015-10-28 21:08:19.067031683 +1100
@@ -36,7 +36,8 @@
 changes of phrasing) have been omitted from the change list. </p>
 
 <p>Text versions of the File are available through a <a
-href="oldversions/index.html">historically appropriate interface</a>.</p>
+href="http://www.catb.org/jargon/oldversions/">historically appropriate
+    interface</a>. [and a <a href="https://bitbucket.org/sinewalker/jargon/src/f39813682526c66f1802b3b1bb9d11203cc7ae65/www.catb.org/_esr/jargon/oldversions/?at=default">file listing</a> on my <a href="https://bitbucket.org/sinewalker/jargon">Bitbucket repository</a> -- M. Lockhart]</p>
 
 <p>You can also download a copy of the Jargon File <a
 href="jargsrc.tar.gz">masters and the production tools</a> used to generate
